src/util/chains.ts

```diff
@@ -358,6 +376,13 @@ export const WRAPPED_NATIVE_CURRENCY: { [chainId in ChainId]: Token } = {
     'WETH',
     'Wrapped Ether'
   ),
+  [ChainId.SEPOLIA]: new Token(
+    11155111,
+    '0xD0dF82dE051244f04BfF3A8bB1f62E1cD39eED92',
```

Comment:
this is the WETH address that we (Uniswap) deployed: 0xfFf9976782d46CC05630D1f6eBAb18b2324d6B14

Reply:
TYSM charlie!